### DISCUSSION TITLE
Polygon-Edge release v0.4.1 chat

### DISCUSSION THREAD

**Discussion by samisbakedham** [0xPolygon/polygon-edge#576] [General]
Hey guys,

Saw the new release (0.4.1) and loved the improvements. Specifically, the addition of eth_getFilterlogs! This method is used often by blockscout, the open-source explorer for EVM compatible chains, and we've been using it to index and debug our chain - Candle which uses polygon edge. Excited to upgrade and try it out!

**Comment by laviniat1996** [comment]
Hello @samisbakedham ,

Thank you very much for the feedback!

We are glad that you loved our new version release for Polygon Edge. It is great that **eth_getFilterlogs** is something that you find helpful and we can't wait for you to try it out. 

Wishing you an amazing week!
